components/centraldashboard/app/api_workgroup.ts

```diff
@@ -39,14 +39,16 @@ interface EnvironmentInfo {
     isClusterAdmin: boolean;
 }
 
-export type SimpleRole = 'owner'| 'contributor';
```

Comment:
This seems a bit concerning that a new role is getting added. What is the purpose/permissions of this role in regards to a namespace?

Reply:
Yes and no, it's a new thing, but we don't use manage contributors anyway, so the places where it would be use are null and void https://github.com/kubeflow/kubeflow/commit/833c64cc1fef2940ed2cfac3658149a7b9d75ca7.
Though yes it will need to be confirmed